<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntiSpamLayerSettingsClass</class>
 <widget class="QScrollArea" name="AntiSpamLayerSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AntiSpamSettings</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>258</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QCheckBox" name="fromClBox">
      <property name="text">
       <string>Accept messages only from
contact list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="notifyBox">
      <property name="text">
       <string>Notify when blocking message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="authorizationBox">
      <property name="text">
       <string>Do not accept NIL messages
concerning authorization</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="urlBox">
      <property name="text">
       <string>Do not accept NIL messages
with URLs</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="botGroupBox">
      <property name="title">
       <string>Enable anti-spam bot</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Question:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="questionTextEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Answer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="answerLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Msg on right answer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="afterAnswerLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="invisibleBox">
         <property name="text">
          <string>Don't send question/reply
if my status is &quot;invisible&quot;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
